HUDKit/Classes/Progress/HUDProgressViewController.swift

```diff
@@ -67,13 +67,16 @@ public class HUDProgressViewController: UIViewController {
         }
     }
 
+    public var blurEffect: UIBlurEffect?
+
     // MARK: - Initializer
 
-    public init(customView: UIView?, status: String?) {
+    public init(customView: UIView?, status: String?, blurEffect: UIBlurEffect? = nil) {
```

Comment:
I think the `blurEffect` type should be `UIVisualEffect` rather than `UIBlurEffect` to receive `UIVibrancyEffect` instances.

e.g.

```swift
public init(customView: UIView?, status: String?, visualEffect: UIVisualEffect? = nil)
```